sound/soc/sof/topology.c

```diff
@@ -2009,9 +1748,6 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	case snd_soc_dapm_demux:
 		ret = sof_widget_parse_tokens(scomp, swidget, tw,  token_list, token_list_size);
 		break;
-	case snd_soc_dapm_effect:
-		ret = sof_widget_load_process(scomp, index, swidget, tw);
-		break;
```

Comment:
also lost on this one

a) why did you move the case before the dapm_pga?
b) why are we adding code?